app/graphql/albums/resolvers.js

```diff
@@ -0,0 +1,26 @@
+const logger = require('../../logger');
+const albumsHelpers = require('../../helpers/albums');
+const albumsService = require('../../services/albums');
+
+exports.getAlbum = (root, args) => {
+  logger.info(`Fetching album with id: ${args.id}`);
+  return albumsService.getAlbum(args.id).then(albumsHelpers.albumMapper);
+};
+
+exports.getAlbums = (root, args) => {
+  const { filter, offset, limit, orderBy } = args;
+  logger.info(`Fetching albums list... offset: ${offset}, limit: ${limit}, orderBy: ${orderBy}`);
+  return albumsService
+    .getAlbums()
+    .then(albums => albumsHelpers.filterAndFormat(albums, { filter, offset, limit, orderBy }));
+};
+
+exports.getPhotos = (root, args) => {
+  const { id } = root ? root : args;
+  logger.info(`Fetching photos of album with id: ${id}`);
+  return albumsService.getPhotos({ albumId: id });
+};
+
+exports.typeResolvers = {
+  photos: exports.getPhotos
```

Comment:
do you need to export all functions  or just resolvers?

Reply:
I guess you say it because of `getPhotos`, the reason I exported `getPhotos` is because I thought that maybe at some point you would want to query photos directly, and in this way you can use the same function